### ISSUE TITLE
Multiple Triggers Per Object

### ISSUE THREAD

**Issue by tmowbrey** [afawcett/declarative-lookup-rollup-summaries#309]
I have already defined triggers on the child objects of used in my roll-ups. I would much rather not have an entirely new trigger created if possible.

Is it possible to do a query on existing triggers for the object and iterate through the lines to see if the line dlrs.RollupService.triggerHandler(); exists rather than enforcing the trigger is created based on trigger name?

**Comment by afawcett** [comment]
Yes that should be possible, though gets complex if trigger patterns are used as the line will not be in the immediate trigger code. I agree though some kind of way to say that the implementor takes on responsibility that the method is called. Perhaps some "Custom Trigger" checkbox which if ticked, allows Activate to be true.

**Comment by aheber** [comment]
This is handled by the currently available developer APIs, put the Rollup into `Developer` mode then in your trigger use

```apex
dlrs.RollupService.rollup(
    Map<Id, SObject> existingRecords,
    Map<Id, SObject> newRecords,
    Schema.SObjectType sObjectType
  );
```

Pass in `Trigger.old` and `Trigger.new` and the SObjectType of the records.
